tests/integrations/instructor/instructor_test.py

```diff
@@ -64,7 +64,7 @@ def test_instructor_openai(
     assert op_name_from_ref(call.op_name) == "openai.chat.completions.create"
     output = call.output
     output_arguments = json.loads(
-        output.choices[0].message.tool_calls[0].function.arguments
+        output["choices"][0]["message"]["tool_calls"][0]["function"]["arguments"]
```

Comment:
why do you have to change from dot to bracket notation?

Reply:
The output here comes from the post processor I'm modifying. In practice `value` can have many types depending on integrations, sdk versions, and I think even the type of completion being requested. In some cases the post processor returns a pydantic object, in some cases it returns a dict. For the standard case where we receive a `ChatCompletion` we will now return a dict with this extra key. The alternative would be adding the attribute to the pydantic class which feels hackier, and ultimately I think the output just gets dumped later in our pipe anyway.